requirements.txt

```diff
@@ -3,5 +3,5 @@ enum34
 piexif
 pika==1.0.0
 pygdal==2.2.2.*
-agpypeline
+agpypeline==0.0.34
```

Comment:
How does this version differ from the develop branch https://github.com/AgPipeline/agpypeline/tree/develop?

Reply:
This version of agpypeline contains pygdal==2.2.2.*, which is used by the Dockerfile but not by the pylint_check.yaml file. The dockerfile uses the 2.2.2.* version of gdal

Reply:
Can you use version 0.0.21 of the library if your changes match up with https://github.com/AgPipeline/agpypeline/pull/8/files? If not, let me know. If it can then version 0.0.21 can become "official" by tagging the PR just listed.

Reply:
We're going with creating an official release of version 0.0.34